src/components/Common/Forms.js

```diff
@@ -71,3 +82,102 @@ export const Fieldset = styled.section`
     margin-bottom: ${remcalc(24)};
   }
 `;
+
+const fadeIn = keyframes`
+  from{
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const SearchField = styled.section`
+  position: relative;
+  margin-bottom: ${remcalc(36)};
+`;
```

Comment:
Is the whole search field design signed off by James? If not should probably ping him to review the design

Reply:
This for now is just an idea for this page, but definitely I will talk with him about this.

Reply:
I was talking with Hannah and it happens that James is no longer work on the site, so any sing off is her responsibility.